packages/fuels-test-helpers/src/wallets_config.rs

```diff
@@ -1,34 +1,40 @@
 // These constants define the default number of wallets to be setup,
 // the number of coins per wallet and the amount per coin
+use fuels_signers::fuel_crypto::fuel_types::AssetId;
+
 pub const DEFAULT_NUM_WALLETS: u64 = 10;
 pub const DEFAULT_NUM_COINS: u64 = 1;
 pub const DEFAULT_COIN_AMOUNT: u64 = 1_000_000_000;
 
+#[derive(Debug, Clone)]
+pub struct AssetsConfig {
```

Comment:
How about calling it `AssetConfig` instead of `AssetsConfig` (notice the `s` gone)? This is the configuration for a single asset.